kinto/plugins/kinto_admin/tests/__init__.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+import webtest
+from kinto.core import utils as core_utils
+from kinto.core import testing as core_support
+
+
+def get_user_headers(user):
+    credentials = "%s:secret" % user
+    authorization = 'Basic {0}'.format(core_utils.encode64(credentials))
+    return {
+        'Authorization': authorization
+    }
+
+
+class BaseWebTest(object):
+    config = '../kinto.ini'
```

Comment:
We could use `kinto.ini` also as you said we might move that in the test folder.